### ISSUE TITLE
Comment at odds with code (loss function)

### ISSUE THREAD

**Issue by emilesilvis** [StatQuest/signa#21]
https://github.com/StatQuest/signa/blob/708bf4d44a3acfb4278ca004e305116d516c0e38/chapter_03/chapter_03_multiple_inputs_and_outputs.ipynb#L870

The comment states that we'll use Cross Entropy to calculate the loss, but `# self.loss = nn.CrossEntropyLoss()` is commented out, and we have `self.loss = nn.MSELoss(reduction='sum')`.

**Comment by StatQuest** [comment]
OK. I removed those comments. We don't use cross entropy until the lesson for the next chapter.
